src/main/java/com/cognitive/nih/niddk/mccapi/controllers/ContactController.java

```diff
@@ -26,12 +28,26 @@
     private final QueryManager queryManager;
     private final IR4Mapper mapper;
 
+    @Value("${mcc.careteam.use.active}")
+    private String useActiveCareplans;
+    boolean isUseActiveCareplans;
```

Comment:
Maybe change to "isUsingActiveCareplans"?

Reply:
Should actually be useActiveCareTeams... will fix all over